### PR TITLE
feat(peg): explain alert history details

### PR DESCRIPTION
**Pull request by chapati23**
## The Problem

- Peg alert history uses internal rule names and policy-state terms that do not explain the incident.
- Operators cannot see the exact threshold, wait time, or provider failure behind an alert.
- UI pull requests do not require a base-versus-head visual comparison.

## The Solution

- Use short, cause-first alert text in the dashboard, Grafana, and Slack.
- Add an accessible Details disclosure with the recorded cause, threshold, and wait time.
- Require verified Before and After screenshots when the ship skill publishes a UI change.

## Visual comparison

Route: `/peg-monitoring` — Recent alerts

Base `c4312d59506c3b126d05815211dc9820dc7d3fa3` → Head `b950a66ae24c8bef9b636a22a5560c74db2802e4`

Viewport: 1440×900. Both images use the same fixed clock and deterministic fixture data.

| Before | After |
| --- | --- |
| Before: existing Peg alert wording | After: cause-first wording with Kraken details expanded |

## Details

- Preserve provider failure reasons and HTTP status evidence through alert history parsing.
- Query a bounded history lead-in so alerts near the seven-day boundary retain their Pending-to-Alerting wait without displaying older rows.
- Keep historical explanations bound to the policy version that produced each alert.
- Keep the repository ship skill and Claude mirror byte-aligned. The machine-global ship skill has the same visual-evidence rule.
- Refs #1870. Terraform apply and production proof remain post-merge work.
- Claude Security was unavailable because the current Claude credit pool is exhausted. A fresh Codex semantic review and the deterministic local review found no actionable issue.

## Validation

- `pnpm agent:quality-gate --run` — all mapped commands passed on final head, including 4,351 dashboard tests, type checks, lint, coverage, Terraform validation, browser tests, React Doctor, and skill-mirror checks.
- Focused alert-explanation regression tests — 16 passed.
- Fresh-context Codex autoreview of the final review-fix batch — clean after one accepted stale-data edge-case fix.
- Browser check at 1440×900 — exact base `c4312d59` and exact head `b950a66a`, fixed clock and fixture data, disclosure interaction verified, no console errors.

## Deferrals

- None

## Checklist

- [x] The Problem has no more than three bullets.
- [x] The Solution is plain English and understandable before reading the diff.
- [x] Deeper implementation details come after the opening two sections.
- [x] Every knowing deferral is listed under Deferrals with a GitHub issue link.
- [x] No architecture decision is required. This change extends the existing alert-history and ship workflows.

---

> [!NOTE]
> **Medium Risk**
> Touches Grafana-backed alert history parsing and operator-facing incident copy; behavior is heavily tested but wrong pairing or policy matching could misstate wait times or thresholds.
> 
> **Overview**
> **Peg alert history** now ships operator-facing copy and richer context end-to-end: Grafana/Terraform notification templates use clearer stale/unavailable wording, the alerts API attaches bounded `evidence` and derives **Pending→Alerting** wait times from an extra seven-day Grafana history lead-in (without showing older rows), and **Recent alerts** adds a **Details** disclosure with policy-aware explanations (thresholds and waits only when the event matches the current decision package).
> 
> The **ship** skill (repo and Claude mirrors) now requires verified before/after screenshots for UI PRs—captured from isolated base vs head worktrees and posted via the GitHub web comment composer.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 5b9c87acc5fd472df9624cb0c7717cdd2c157633. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

**Comment by chapati23** [comment]
## Visual comparison

Route: `/peg-monitoring` — Recent alerts

Base `c4312d59506c3b126d05815211dc9820dc7d3fa3` → Head `5b9c87acc5fd472df9624cb0c7717cdd2c157633`

Viewport: 1440×900. Both images use the same fixed clock and deterministic fixture data.

| Before | After |
| --- | --- |
| Before: existing Peg alert wording | After: cause-first wording with Kraken details expanded |

File: ui-dashboard/src/app/peg-monitoring/_lib/peg-alert-explanation.ts [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    return stalePriceExplanation(event, policy);
+  }
+  const source = event.evidence.sourceName;
+  const market = marketName(event);
+  const explanation =
+    SOURCE_FAILURE_EXPLANATIONS[event.evidence.failureReason ?? 0];
+  return explanation === undefined
+    ? unknownSellPriceExplanation(event, policy)
+    : `${explanation(source, market, event.severity === "cleared")}${pendingSentence(event)}`;
+}
```
[the remark this answers is not in the document]
Comment (chapati23) [reply ui-dashboard/src/app/peg-monitoring/_lib/peg-alert-explanation.ts:134]: Fixed in 2dfb6ce0 — classified blind failures now include the consecutive-check threshold, including stale-data failures.
